src/containers/Profiles/EditProfileContainer.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { 
+    fetchUserProfile, 
+    requestEditProfile 
+} from '../../store/actions/async/profiles';
+import EditProfile from '../../components/Profiles/EditProfile';
+import { dismissMessage } from '../../store/actions/sync/profiles';
+import PropTypes from 'prop-types';
+
+export class EditProfileContainer extends Component {
+    state = {
+        firstName: "",
+        lastName: "",
+        bio: "",
+        image: null,
+        disabled: true
+    };
+
+    componentDidMount = () => {
+        const { username } = this.props.match.params;
```

Comment:
`username` and `match` missing in prop validation. 
We can also use destructuring like:
` const { match: { params: {username} } = { params: {} } } = this.props;`

Use destructuring to get items on `this.props`. `this.props` is duplicated in this file. Make the appropriate changes.